linux.cc

```diff
@@ -396,6 +397,22 @@ static long sys_getcpu(unsigned int *cpu, unsigned int *node, void *tcache)
     return 0;
 }
 
+#define __NR_sys_prlimit __NR_prlimit64
+static int sys_prlimit(pid_t pid, int resource, const struct rlimit *new_limit,
+    struct rlimit *old_limit)
+{
+    if (new_limit) {
+        return 0;
+    }
+
+    if (old_limit) {
+        return getrlimit(resource, old_limit);
```

Comment:
Probably need to verify first that "pid" is 0 or the current pid or something, and if not return an error (?)

Reply:
Will fix it in a new PR.